bazel/core.bzl

```diff
@@ -71,10 +71,11 @@ MLN_GENERATED_OPENGL_SHADER_HEADERS = [
     "include/mbgl/shaders/gl/drawable_debug.hpp",
     "include/mbgl/shaders/gl/drawable_fill.hpp",
     "include/mbgl/shaders/gl/drawable_fill_outline.hpp",
+    "include/mbgl/shaders/gl/drawable_fill_pattern.hpp",
     "include/mbgl/shaders/gl/drawable_fill_outline_pattern.hpp",
+    "include/mbgl/shaders/gl/drawable_fill_outline_triangulated.hpp",
```

Comment:
Don't forget to update CMakeLists.txt to reflect this change

Reply:
Done 👍